app/renderer/js/components/handle-external-link.ts

```diff
@@ -8,10 +8,8 @@ const { shell, app } = remote;
 
 const dingSound = new Audio('../resources/sounds/ding.ogg');
 
-// TODO: TypeScript - Figure out a way to pass correct type here.
-function handleExternalLink(this: any, event: any): void {
-	const { url } = event;
```

Comment:
I'm guessing it's a legacy code which we forgot to remove?

Reply:
Yes, typescript detected that here, and it is passed correctly now.